dotcom-rendering/src/components/Card/components/PlayIcon.tsx

```diff
@@ -71,12 +83,14 @@ const getIconSizeOnMobile = (imagePositionOnMobile: ImagePositionType) =>
 export const PlayIcon = ({
 	imageSize,
 	imagePositionOnMobile,
+	isPlayableMediaCard,
 }: {
 	imageSize: ImageSizeType;
 	imagePositionOnMobile: ImagePositionType;
+	isPlayableMediaCard: boolean;
 }) => {
 	return (
-		<div css={iconWrapperStyles}>
+		<div css={[iconWrapperStyles, isPlayableMediaCard && iconPulseStyles]}>
```

Comment:
Do we need the triple nesting of an `svg` inside a `span` inside a `div`, here? Can be addressed in a follow-up.